terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1061,15 +1061,15 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                                --volume /mnt/gitlab/config:/etc/gitlab \
                                --volume /mnt/gitlab/logs:/var/log/gitlab \
                                --volume /mnt/gitlab/data:/var/opt/gitlab \
-                               gitlab/gitlab-ce:14.5.2-ce.0
+                               gitlab/gitlab-ce:14.6.1-ce.0
```

Comment:
The previous version was 14.6.0. I forgot to open a PR when I updated GitLab from 14.5.2 to 14.6.0.